### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by cardistack**
Changes to some wording and re-titling 
Title seen in browser when multiple tabs are open.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 go-f2lb-website
 
 vendor
-
+node_modules
```
Comment (safanaj) [inline .gitignore]: nodejs related files should be not in the top dir but only under webui, also files like package.json and yarn.lock are meaningful to be tracked under git.

File: webui/src/routes/+page.svelte [hunk 2]
```diff
@@ -14,7 +14,7 @@
 </script>
 
 <svelte:head>
-<title>Welcome</title>
+<title>Welcome to F2LB Queue Manager</title>
```
Comment (safanaj) [inline webui/src/routes/+page.svelte:17]: I would not define this website/page a "Manager" but I am ok with any title

File: webui/package.json [hunk 3]
```diff
@@ -37,5 +37,8 @@
     "vite-plugin-top-level-await": "^1.1.1",
     "vite-plugin-wasm": "^2.1.0"
   },
-  "type": "module"
+  "type": "module",
```
Comment (safanaj) [inline webui/package.json]: I am not a nodejs expert but I think that webui is not a dependency of the webui package itself

**Comment by safanaj** [comment]
Regarding how to build we have to improve the doc with some example but usually I just do:
```
make clean-all build
```
For the dependencies I installed them manually, getting the binaries where available or building them from sources.

**Review by safanaj — CHANGES_REQUESTED** [review]

File: webui/package.json (hunk 3 above)
Comment (cardistack) [reply webui/package.json]: sorry this shouldn't be in PR

File: .gitignore (hunk 1 above)
Comment (cardistack) [reply .gitignore]: Sorry this shouldn't be in PR

**Review by tahussle — COMMENTED** [review]
More slight formatting changes

Added option not to set -mod=vendor option in makefile as  built binary was not embeding webui/build/_app for some reason .
Set source code link frontend to github

**Review by safanaj — APPROVED** [review]

**Pull request merged**